packages/react/src/callout/docs/overview.mdx

```diff
@@ -96,3 +96,19 @@ The feature variant of the callout can be used to draw even more emphasis to the
 	<Text as="p">Description of the callout.</Text>
 </Callout>
 ```
+
+## Title
+
+By default, `Callout` renders `CalloutTitle` as an `h2`. If you need to change the title’s semantic tag to an `h3`, for instance, you can use the `CalloutTitle` component in the `title` prop to explicitly set it. For information on maintaining proper heading levels refer to [Structuring headings and labels](/content/content-structure#structuring-headings-and-labels).
```

Comment:
I've always wondered, should this be `a h2` since `h` stands for `heading`?

Reply:
You read "h" not "heading" and I'll be damned if I'm ever going to pronounce h as haitch (which would work better with "a")

Reply:
>I'll be damned if I'm ever going to pronounce h as haitch

Haha you and me both, buddy!